tests/bluetooth/host/mock_host/src/net.c

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
can you rename that file to `buf.c` ?

Reply:
These definitions exist in subsys/net/buf.c.
I moved it to net/buf.c.